### PR TITLE
add info for usage and solo param

### PR DESCRIPTION
**Pull request by sjmiles**

### CODE REVIEW

File: shells/pipes-shell/README.md [hunk 1]
```diff
@@ -35,6 +58,10 @@ pipes-shell/web/index.html?log[=[0..2]]&remote-explore-key=[key]
 - remote-explore-key
   - used to connect to remote devtools (aka Arcs Explorer)
 
+- solo
+  - fetch manifest from [mainfest-url] instead of the default
```
Comment (shaper) [inline shells/pipes-shell/README.md]: mainfest -> manifest

File: shells/pipes-shell/README.md [hunk 2]
```diff
@@ -20,9 +20,32 @@ Also inside each platform folder is a deploy folder that contains tools for buil
...
+```
+  window.DeviceClient = {
+    shellReady() {
+      console.warn('context is ready!');
+    },
+    foundSuggestions(json) {
+    }
+  };
+```
+Also, when run under headful chrome, clicking the display will run a test recieve.
```
Comment (shaper) [inline shells/pipes-shell/README.md]: recieve -> receive

File: shells/pipes-shell/README.md [hunk 3]
```diff
@@ -20,9 +20,32 @@ Also inside each platform folder is a deploy folder that contains tools for buil
...
 
-pipes-shell/web/index.html?log[=[0..2]]&remote-explore-key=[key]
+pipes-shell exposes JS entry/exit points designed to be bound into another process. These entry points can also be exercised via console. Here are some examples:
+
+```
+> ShellApi.observeEntity(`{"type": "address", "name": "East Mumbleton"}`)
+> ShellApi.receiveEntity(`{"type": "com.google.android.apps.maps"}`)
+> ShellApi.receiveEntity(`{"type": "com.music.spotify"}`)
+```
+Results are returned via `DeviceClient.foundSuggestions(json)` (if it exists)
```
Comment (shaper) [inline shells/pipes-shell/README.md]: Give example actual json that might be passed back? Or, explain where a reader could look to learn more about it.

Comment (sjmiles) [reply shells/pipes-shell/README.md]: Done, thanks for the proof-read.

**Pull request merged**